tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,338 @@
+"""Tests for the Incremental Update feature"""
+
+from datetime import date, timedelta
+from fauxfactory import gen_alpha
+from nailgun import entity_mixins
+from nailgun.entities import (
+    ActivationKey, ContentView, ContentViewFilterRule, ContentViewVersion,
+    Errata, ErratumContentViewFilter, HostCollection, LifecycleEnvironment,
+    Organization, Repository, Subscription, System
+)
+from robottelo.api.utils import (
+    enable_rhrepo_and_fetchid, promote, upload_manifest
+)
+from robottelo.cli.contentview import ContentView as ContentViewCLI
+from robottelo.common import manifests
+from robottelo.common.decorators import run_only_on, skip_if_bug_open
+from robottelo.vm import VirtualMachine
+from unittest import TestCase
+
+PRD_SETS = {
+    'rhel_66': {
+        'product': u'Red Hat Enterprise Linux Server',
+        'reposet': u'Red Hat Enterprise Linux 6 Server (RPMs)',
+        'reponame': u'Red Hat Enterprise Linux 6 Server RPMs x86_64 6.7',
+        'arch': u'x86_64',
+        'releasever': u'6.7',
+        'label': u'rhel-6-server-rpms'
+    },
+    'rhel6_sat6tools': {
+        'product': u'Red Hat Enterprise Linux Server',
+        'reposet': u'Red Hat Satellite Tools 6.1 (for RHEL 6 Server) '
+                   '(RPMs)',
+        'reponame': u'Red Hat Satellite Tools 6.1 for RHEL 6 Server RPMs '
+                    'x86_64',
+        'arch': u'x86_64',
+        'releasever': None,
+        'label': u'rhel-6-server-satellite-tools-6.1-rpms'
+    }
+}
+
+
+@run_only_on('sat')
+class TestIncrementalUpdate(TestCase):
+    """Tests for the Incremental Update feature"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Creates all the pre-requisites for the Incremental updates test"""
+        # Step 1 - Create a new Organization
+        cls.org = Organization(name=gen_alpha(length=5)).create()
```

Comment:
any special reason to use 5 as length? Seems that using just the default would be ok and will make the line clean.